src/copas.lua

```diff
@@ -597,15 +597,18 @@ function copas.addserver(server, handler, timeout)
     end
 end
 
-function copas.removeserver(server)
+function copas.removeserver(server, close_it)
```

Comment:
Wouldn't it be more intuitive to call it `no_close` and pass `true`? Having to pass flags as `false` seems odd